devices.js

```diff
@@ -3091,6 +3091,26 @@ const devices = [
             await configureReporting.batteryPercentageRemaining(endpoint);
         },
     },
+    {
+        zigbeeModel: ['DWS003'],
+        model: 'DWS003',
+        vendor: 'Hive',
+        description: 'Contact sensor',
+        supports: 'contact, temperature, battery',
+        fromZigbee: [
+            fz.temperature, fz.ias_contact_alarm_1, fz.battery
```

Comment:
build is failing because it wants this line to end in a comma btw

Reply:
good spot, thanks. BTW - your work on MOT003 made adding this a quick win!

Reply:
No worries at all dude :)